docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+
+## Configure a deployment manifest for the nested IoT Edge
+
+A deployment manifest is a JSON document that describes which modules to deploy to the IoT Edge device. For more information about how deployment manifests work and how to create them, see [Understand how IoT Edge modules can be used, configured, and reused](../iot-edge/module-composition.md).
+
+To deploy the connected registry module using the Azure CLI, save the following deployment manifest locally as a `.json` file. 
+
+[!IMPORTANT] In the bellow deployment manifest, the IP address `10.16.7.4` is the IP address of the device hosting parent connected registry. Make sure you replace this IP address with the one your parent device uses.
+
+```json
+{
+    "modulesContent": {
+        "$edgeAgent": {
+            "properties.desired": {
+                "modules": {
+                    "connected-registry": {
+                        "settings": {
+                            "image": "10.16.7.4/acr/connected-registry:0.2.0",
+                            "createOptions": "{\"HostConfig\":{\"Binds\":[\"/home/azureuser/connected-registry:/var/acr/data\",,\"/usr/local/share/ca-certificates:/usr/local/share/ca-certificates\",\"/etc/ssl/certs:/etc/ssl/certs\"]}}"
+                        },
+                        "type": "docker",
+                        "env": {
+                            "ACR_REGISTRY_CONNECTION_STRING": {
+                                "value": "ConnectedRegistryName=myconnectedmirror;SyncTokenName=myconnectedmirror-sync-token;SyncTokenPassword=s0meCoMPL3xP4$$W0rd001!@#;ParentGatewayEndpoint=10.16.7.4;ParentEndpointProtocol=https"
+                            }
+                        },
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "version": "1.0"
+                    },
+                    "IoTEdgeApiProxy": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-api-proxy:latest",
+                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"443/tcp\": [{\"HostPort\": \"443\"}]}}}"
+                        },
+                        "type": "docker",
+                        "version": "1.0",
+                        "env": {
+                            "NGINX_DEFAULT_PORT": {
+                                "value": "443"
+                            },
+                            "CONNECTED_ACR_ROUTE_ADDRESS": {
+                                "value": "connectedRegistry:8080"
+                            },
+                            "NGINX_CONFIG_ENV_VAR_LIST": {
+                                    "value": "NGINX_DEFAULT_PORT,BLOB_UPLOAD_ROUTE_ADDRESS,CONNECTED_ACR_ROUTE_ADDRESS,IOTEDGE_PARENTHOSTNAME,DOCKER_REQUEST_ROUTE_ADDRESS"
+                            },
+                            "DOCKER_REQUEST_ROUTE_ADDRESS": {
```

Comment:
Is it supposed to be same as the value of `CONNECTED_ACR_ROUTE_ADDRESS`?

Reply:
DOCKER_REQUEST_ROUTE_ADDRESS is the env for open registry.

Reply:
@huanwu do you mean `DOCKER_REQUEST_ROUTE_ADDRESS` needs to be configured here to `registry:5000`?

Reply:
I double verified and think we can remove this env if we remove this path from ngnix config.